pages/common/pygmentize.md

```diff
@@ -1,27 +1,27 @@
 # pygmentize
 
-> Python based syntax highlighter.
+> Python-based syntax highlighter.
 
-- Highlight file syntax and print to standard output. Language is inferred from the file extension:
+- Highlight file syntax and print to standard output (language is inferred from the file extension):
 
 `pygmentize {{file.py}}`
 
 - Highlight syntax for a given language:
```

Comment:
Perhaps "Explicitly set the language when syntax highlighting" or something would be more understandable?